.github/workflows/windows_cmake.yml

```diff
@@ -0,0 +1,160 @@
+name: Windows CMake build
+
+on:
+  push:
+    branches: [ RC_1_2 ]
+  pull_request:
+    types: [edited, opened, reopened, synchronize]
+    branches: [ RC_1_2 ]
+
+env:
+  # openssl: 1.1.1g#1
+  # boost: 1.73.0
+  VCPKG_COMMIT: 27a2418e91179d8607862348d7b498558e98a0ab
+  VCPKG_DEST_WIN: C:\libt_tools\vcpkg
+  VCPKG_TRIPLET: "x64-windows"
+  BUILD_VARIANT: "shared"
+  LIBT_STATIC_RT: "OFF"
+  LIBT_BSL: "ON"
+  LIBT_TESTS: "ON"
+  LIBT_DEPR_FUN: "ON"
+  LIBT_PY_BINDINGS: "ON"
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+
+  ci_windows_cmake:
+    name: win cmake
+
+    strategy:
+      matrix:
+        build_config: [rel, dbg]
+        build_variant: [shared, static]
+        deprecated_functions: [depr, nodepr]
+        build_tests: [tests, notests]
+        python_bindings: [py, nopy]
```

Comment:
it seems a bit excessive to include `python_bindings` and `build_tests` in the build matrix. It's not clear to me that there's any value in building all configurations with *and* without tests. I suppose when building without tests `TORRENT_EXTRA_EXPORT` is not defined, but twice as many build configuration is a lot. It would seem reasonable to me to always build the tests.

Likewise with python. I think building the python binding doesn't affect any other aspect of the library. It could just always be on I think.